packages/app/scripts/start.js

```diff
@@ -95,21 +95,19 @@ function setupCompiler(port, protocol) {
     // We use stats.toJson({}, true) to make output more compact and readable:
     // https://github.com/facebookincubator/create-react-app/issues/401#issuecomment-238291901
     var json = stats.toJson({}, true);
-    var formattedErrors = json.errors.map(
-      message => 'Error in ' + formatMessage(message)
-    );
+    var formattedErrors = json.errors
```

Comment:
It is because errors or not only at the root of json. The children can also have errors. This is where I got stunned a bit trying to figure out some type errors I believe it was. It was located on the children error. So yeah. It is still the same, just reduces in the children errors as well :)